lib/membrane/srtp/encryptor.ex

```diff
@@ -13,17 +13,28 @@ defmodule Membrane.SRTP.Encryptor do
 
   def_options policies: [
                 spec: [ExLibSRTP.Policy.t()],
+                default: [],
                 description: """
                 List of SRTP policies to use for encrypting packets.
                 See `t:ExLibSRTP.Policy.t/0` for details.
                 """
               ]
 
   @impl true
-  def handle_init(options) do
-    {:ok, Map.from_struct(options) |> Map.merge(%{srtp: nil})}
+  def handle_init(%__MODULE__{policies: policies}) do
+    state = %{
+      policies: policies,
+      srtp: nil,
+      ready: ready?(policies),
+      buffered_demand: []
```

Comment:
that's not necessary, see https://hexdocs.pm/membrane_core/Membrane.Element.Action.html#t:redemand_t/0